test/ComposerRequireCheckerTest/JsonLoaderTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace ComposerRequireCheckerTest;
+
+use ComposerRequireChecker\JsonLoader;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers \ComposerRequireChecker\JsonLoader
+ */
+class JsonLoaderTest extends TestCase
+{
+
+    /**
+     * @expectedException \ComposerRequireChecker\Exception\NotReadableException
```

Comment:
Note that the `@expectedException` is to be avoided, while `->expectException()` should be placed right before the call causing the actual exception